JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1437,82 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.XbarchartNoId <- function(jaspResults, dataset, options) {
+
+  ready <- (length(options$variables) > 1)
+  if (!ready)
+    return()
+
+  jaspResults[["XbarPlot"]] <- createJaspPlot(title = "X bar chart", width = 650, aspectRatio = 1)
+  jaspResults[["XbarPlot"]]$dependOn(c("Xbarchart", "variables"))
+  jaspResults[["XbarPlot"]]$position <- 11
+  XbarPlot <- jaspResults[["XbarPlot"]]
+
+  data1= na.omit(as.data.frame(dataset))
```

Comment:
```suggestion
  data1 <- na.omit(dataset)
```
please use `<-` consistently for assignment instead of `=`. Also, the `as.data.frame` here is unnecessary because `dataset` is always a data.frame.